### ISSUE TITLE
Documentation request: Bare metal setup, Design overview, failover, operating experience

### ISSUE THREAD

**Issue by reachbach** [arachnys/cabot#118]
Of all the documented tools that monitor against thresholds based on Graphite, I found Cabot's approach the most comprehensive (with a service centric view) and it's problem focus correct with respect to monitoring against graphite. However, while attempting a test setup for evaluation, I was stalled due to lack of the following:

1) Setup documentation (at http://cabotapp.com/qs/quickstart.html) for standalone bare metal environments. (The cabot-docker project doesn't appear ready for production environments.)

2) Design overview explaining how the database, redis etc mentioned in the dependencies are actually used.

3) Documentation around failover / restoration considerations for Cabot itself.

4) Mention of duration of use in production and observed functional correctness (e.g. any missed notifications?), stability, performance, scale (w.r.t number of metrics and frequency of sampling) in a production environment.

Can the above be included in the documentation to help prospective users please? Thanks. And kudos again for a good contribution to the community.

**Comment by tommiatplayfish** [comment]
So... digitalocean instructions are mentioned for basic guideline for bare metal installation... but I can't find them :) Any pointers would be helpful.

Also the cabot-docker doesn't work any more it seems, the cabot image fails with:
  File "/cabot/cabot/settings.py", line 133, in <module>
    EMAIL_PORT = int(os.environ.get('SES_PORT', 25))
ValueError: invalid literal for int() with base 10: ''

When I've definitely defined it. Anyways the docker setup is not really documented at all what needs to be configured so it's a bit difficult to debug as containers are what they are.

**Comment by xinity** [comment]
i'm back from the dead !!!!! :(

will PR a draft soon :)

sorry been awefully busy (i'm a father :))

**Comment by dbuxton** [comment]
Congratulations!

**Comment by grzegorz-dubicki** [comment]
Congratulations :smile: :baby: !

**Comment by JackLeo** [comment]
Yep, I completely agree with point that deployment script is not a proper way to distribute/document the tool.

What I did was a git clone, and then followed fabric and bin/ scripts. Also edited setup.py to remove postgres connector and used mysql. Hardest part is to sort out compressor.

if I'll find some time I could do PR for documentation on step by step.

**Comment by i19** [comment]
+1
